hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -612,3 +612,5 @@ distributionset.not.existsDistribution=Distribution set does not exist. Maybe th
 distributionsets.not.exists=Distribution sets do not exists. Maybe the sets were deleted.
 
 targettag.not.exists=Target tag {0} does not exists. Maybe the target tag was deleted.
+
+textfield.min.length = Please enter at least one character.
```

Comment:
I know there is currently no clear pattern how the message keys are defined, but there might be already some validation i18n keys - maybe we could prefix these kind of validation texts  and add `validation` or `validator` or something in the key?